### DISCUSSION TITLE
How to extend my user model and how to use my userModel in a new model?

### DISCUSSION THREAD

**Discussion by Tristannos** [serverpod/serverpod#720] [Q&A] [answered]
I want to add more fields to my user model how do i do that?
Like: Licence number, Location....

And i want to reference to a user model in another model like this:

Name of the class to generate.
class: Rally
table: rally
fields:
  championship_ids: List<int>?
  name: String
  **organisator: User?**
`

**Comment by vlidholt** [comment, marked as answer]
You can reference another class like you are doing above. Serverpod serialization doesn't support inheritance yet, it's possible we may add that in the future. So you will instead have to work with composition (like in your example above).
